packages/authorization-process/src/services/authorizationService.ts

```diff
@@ -77,6 +77,37 @@ export function authorizationServiceBuilder(
         toCreateEventClientAdded(client, correlationId)
       );
 
+      return {
+        client,
+        showUsers: true,
+      };
+    },
+    async createApiClient(
+      clientSeed: ApiClientSeed,
+      organizationId: TenantId,
+      correlationId: string,
+      logger: Logger
+    ): Promise<{ client: Client; showUsers: boolean }> {
+      logger.info(
+        `Creating API client ${clientSeed.name} for consumer ${organizationId}"`
+      );
+      const client: Client = {
+        id: generateId(),
+        consumerId: organizationId,
+        name: clientSeed.name,
+        purposes: [],
+        description: clientSeed.description,
+        relationships: [],
```

Comment:
```suggestion
```

Reply:
As @galales suggested we have to remove `relationships` from the Client's model